spec/controllers/twitter/callbacks_controller_spec.rb

```diff
@@ -32,6 +38,7 @@
       allow(webhook_service).to receive(:perform).and_raise StandardError
       get twitter_callback_url
       account.reload
+      # expect(Avatar::AvatarFromUrlJob).not_to receive(:perform_later).with(account.inboxes.last, '')
```

Comment:
This shouldn't be commented out ? 
We should also add  avatar job expectation for the inbox creation condition